pkg/project/version_bundle.go

```diff
@@ -15,27 +13,11 @@ func NewVersionBundle() versionbundle.Bundle {
 				Kind:        versionbundle.KindFixed,
 				URLs:        []string{"https://github.com/giantswarm/azure-operator/pull/679"},
 			},
-		},
-		Components: []versionbundle.Component{
-			{
-				Name:    "calico",
-				Version: "3.10.1",
-			},
-			{
-				Name:    "containerlinux",
-				Version: key.CoreosVersion,
-			},
 			{
-				Name:    "docker",
-				Version: "18.06.1",
-			},
-			{
-				Name:    "etcd",
-				Version: "3.3.17",
-			},
-			{
-				Name:    "kubernetes",
-				Version: "1.16.7",
+				Component:   Name(),
```

Comment:
You don't need these changelogs at all anymore.

Reply:
See fancy changelog md in aws-operator: https://github.com/giantswarm/aws-operator/blob/master/CHANGELOG.md

Reply:
Ah, right. :+1: